### PR TITLE
Tooltip

### PR DESCRIPTION
**Pull request by WeissSa**
## Pull Request Summary
Added an optional tooltip + text for the arrow buttons

### Change types

- [x] Non-breaking change
- [x] Adding tests
- [x] Front end change

### Testing
- [x] I have added testing to this PR 

### How to manually test this PR (QA instructions)
Go to analysis page and hover over enabled arrows

### CODE REVIEW

**Review by KayleeLChan — APPROVED** [review]
She is giving everything that needed to be gave. 💃 💃 💃 code almost as yummy as asphalt

File: front-end/src/components/IntentLister.jsx [hunk 1]
```diff
@@ -144,25 +144,27 @@ class IntentLister extends Component {
...
-                <div className="fixed left-4 bottom-8">
+                <div className="fixed left-6 bottom-8">
                     <BaseButton
                         click={this.decrementIndex}
                         isDisabled={this.checkIfMinIndex()}
                         icon={{
                             name: this.state.keysHeld["Control"] ? "skip-left" : "arrow-left",
                             size: "40"
                         }}
+                        tooltip="Click this button or press the ← key while holding control to skip to the beginning"
```
Comment (williamyao27) [inline front-end/src/components/IntentLister.jsx:155]: Can we call control "CTRL" or some other thing to indicate that it is a key name? idk might be easier on first glance (same ideas as the left arrow icon vs saying "Left Arrow")

Comment (WeissSa) [reply front-end/src/components/IntentLister.jsx:155]: Great idea

**Review by williamyao27 — APPROVED** [review]
PR #50 🥳

**Pull request merged**